src/ts/components/Match.tsx

```diff
@@ -32,6 +38,14 @@ class Match<TMatch extends IMatch> extends Component<IMatchProps<TMatch>> {
                 matchId={matchId}
                 suggestions={suggestionsToRender}
               />
+              <div className="MatchWidget__feedbackLink">
+                <a 
+                  target="_blank"
+                  href={getFeedbackLink(feedbackInfo)}
+                >
+                  Something's not right? Tell us!
```

Comment:
Maybe?

```suggestion
                  Something not right with this suggestion? Tell us!
```

Reply:
This text is from the UI design.  I'm not changing it.